<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="5">
     <widget class="QPushButton" name="trigonom_2">
      <property name="text">
       <string>show trigonom</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>1) real = </string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="real_1"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QLabel" name="op2_print">
      <property name="text">
       <string>2)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="trigono_1">
      <property name="text">
       <string>show trigonom</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="imagine_1"/>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="algebrs_3">
      <property name="text">
       <string>show algebra</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="div">
      <property name="text">
       <string>div</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>imagine=</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="imagine_2"/>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="set_1">
      <property name="text">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="trigonom_3">
      <property name="text">
       <string>show trigonom</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="real_2"/>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="res_print">
      <property name="text">
       <string>res)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="op1_print">
      <property name="text">
       <string>1)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>2) real =</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="add">
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>imagine=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="n_1"/>
      </item>
      <item>
       <widget class="QPushButton" name="pow_1">
        <property name="text">
         <string>pow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="root_1">
        <property name="text">
         <string>root</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>150</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="algebra_1">
      <property name="text">
       <string>show algebra</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="n_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="pow_2">
        <property name="text">
         <string>pow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="root_2">
        <property name="text">
         <string>root</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="set_2">
      <property name="text">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="mul">
      <property name="text">
       <string>mul</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="sub">
      <property name="text">
       <string>sub</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="algebra_2">
      <property name="text">
       <string>show algebra</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="compare">
      <property name="text">
       <string>compare</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QLabel" name="is_equel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
